Nitrox.Assets.Subnautica/LanguageFiles/it.json

```diff
@@ -1,68 +1,69 @@
-    "Nitrox_FirewallInterfering": "Sembrerebbe che il firewall stia interferendo",
-    "Nitrox_HideIp": "Nascondi gli indirizzi IP",
-    "Nitrox_HidePing": "Nascondi ping",
-    "Nitrox_InvalidUserName": "Inserire un nome di giocatore valido.\n\nNon può contenere spazi o caratteri dubbi.\nCaratteri consentiti: A-Z a-z 0-9 _ . -\nLunghezza: [3, 25]",
-    "Nitrox_Join": "Entra",
-    "Nitrox_JoinServerFailed": "Impossibile entrare nel server",
-    "Nitrox_JoinServerIpAddress": "Indirizzo IP",
-    "Nitrox_JoinServerPassword": "Password:",
-    "Nitrox_JoinServerPasswordHeader": "Richiesta password",
-    "Nitrox_JoiningSession": "Sto entrando nella sessione Multigiocatore",
-    "Nitrox_Kick": "Espelli {PLAYER}",
-    "Nitrox_KickQuestion": "Espellere {PLAYER}?",
-    "Nitrox_LaunchGame": "Caricamento…",
-    "Nitrox_LoadingScreenWarn": "Aspettati problemi che potrebbero compromettere l'esperienza di gioco",
-    "Nitrox_LostConnection": "Connessione al server persa",
-    "Nitrox_LostConnectionWarn": "Non più connesso al server",
-    "Nitrox_Multiplayer": "Multigiocatore",
-    "Nitrox_MultiplayerLoaded": "Client multigiocatore caricato…",
-    "Nitrox_Mute": "Muta {PLAYER}",
-    "Nitrox_MuteQuestion": "Muta {PLAYER}?",
-    "Nitrox_MutedPlayer": "Mutato {PLAYER}",
-    "Nitrox_PlayerColor": "Colore giocatore",
-    "Nitrox_PlayerKicked": "Sei stato espulso dal server",
-    "Nitrox_PlayerListTabName": "Elenco giocatori",
-    "Nitrox_RejectedSessionPolicy": "Richiesta rifiutata…",
-    "Nitrox_RequestingSessionPolicy": "Richiesta di informazioni sui criteri di sessione…",
-    "Nitrox_ServerStopped": "Il server è stato fermato",
-    "Nitrox_ShowPing": "Mostra ping",
-    "Nitrox_SilenceChat": "Disattiva la discussione",
-    "Nitrox_SilencedChatNotif": "La chat è ora silenziata",
-    "Nitrox_StartServer": "Per entrare nel tuo server locale devi prima avviarlo da Nitrox Launcher",
-    "Nitrox_StreamerSettings": "Modalità streamer",
-    "Nitrox_SubmitPassword": "Conferma",
-    "Nitrox_SyncingWorld": "Sincronizzazione del mondo multiplayer…",
-    "Nitrox_TeleportTo": "Teletrasporta su {PLAYER}",
-    "Nitrox_TeleportToMe": "Teletrasporta {PLAYER} da me",
-    "Nitrox_TeleportToMeQuestion": "Teletrasportare {PLAYER} da me ?",
-    "Nitrox_TeleportToQuestion": "Teletrasportarsi su {PLAYER}?",
-    "Nitrox_ThankForPlaying": "Grazie per aver scelto Nitrox!",
-    "Nitrox_UnableToConnect": "Impossibile contattare il server:",
-    "Nitrox_Unmute": "Non mutato {PLAYER}",
-    "Nitrox_UnmuteQuestion": "Non mutare {PLAYER}?",
-    "Nitrox_UnmutedPlayer": "Non mutato {PLAYER}",
-    "Nitrox_Waiting": "In attesa in coda di iscrizione…",
-    "Nitrox_WaitingPassword": "In attesa della password per il server…",
-    "Nitrox_WaitingUserInput": "In attesa dell'inserimento dal parte dell'utente…",
-    "Nitrox_WorldSettling": "In attesa impostazioni del mondo…"
+  "Nitrox_AddServer": "Aggiungi un server",
```

Comment:
No need to add translation to italian, as I said our translation website will check the english one to automatically give people the opportunity to translate.